specs.js

```diff
@@ -1,11 +1,33 @@
 /* global Given, Then, describe */
```

Comment:
Personal preference toward consistency with other projects, I wouldn't mind seeing the tests moved into a tests directory